org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XMLFormattingOptions.java

```diff
@@ -41,6 +41,8 @@ public class XMLFormattingOptions extends org.eclipse.lemminx.settings.LSPFormat
 
 	public static final int DEFAULT_SPLIT_ATTRIBUTES_INDENT_SIZE = 2;
 
+	public static final String DEFAULT_XSI_SCHEMA_LOCATION_SPLIT = "onPair";
```

Comment:
Consider using `XSISchemaLocationSplit.onPair.name()`, so that if we ever decide to rename this setting, this will continue to work.